net/yaaw/Makefile

```diff
@@ -0,0 +1,52 @@
+#
+# Copyright (C) 2014-2016 nanpuyue <nanpuyue@gmail.com>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=yaaw
+PKG_VERSION:=2016-05-28
+PKG_RELEASE:=1
+PKG_MAINTAINER:=nanpuyue <nanpuyue@gmail.com>
```

Comment:
Hmm, if you are going to maintain this package, then `PKG_MAINTAINER` needs to be your REAL name.